src/main/java/spark/route/RouteOverview.java

```diff
@@ -0,0 +1,114 @@
+package spark.route;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import spark.FilterImpl;
+import spark.Request;
+import spark.Response;
+import spark.RouteImpl;
+import sun.reflect.ConstantPool;
+
+import static java.util.Collections.singletonList;
+import static spark.Spark.get;
+
+public class RouteOverview {
+
+    /**
+     * Enables a route overview (showing all the mapped routes)
+     * The overview is made available at "/debug/routeOverview/"
+     */
+    public static void enableRouteOverview() {
```

Comment:
ren to just `enable` and this can be used without static imports and still not getting redundancy in "naming". Ie:
`RouteOverview.enable()`